sound/soc/sof/ipc4-topology.c

```diff
@@ -379,6 +379,39 @@ sof_ipc4_get_input_pin_audio_fmt(struct snd_sof_widget *swidget, int pin_index)
 	return NULL;
 }
 
+static void
+sof_ipc4_evaluate_params_change(struct sof_ipc4_available_audio_format *available_fmt)
```

Comment:
@ujfalusi @ranj063 I'm assuming we can inject this params data into the module creation IPC update when its ready ?

Reply:
Probably a question for @ranj063